keps/sig-node/4815-dra-partitionable-devices/README.md

```diff
@@ -92,9 +101,14 @@ allocated, rather than requiring them to be created before.
 
 ## Motivation
 
-One of the primary motivating examples for supporting partitionable devices
-with DRA is to enable the dynamic allocation of Multi-Instance GPUs
-(MIG) on NVIDIA hardware. MIG devices are represented as fixed-size partitions
+We have several motivating examples for supporting partitionable devices
+with DRA, with the first two described in detail in this document:
```

Comment:
s/in this document:/in this document. Additional details of the other use cases can be found in ...link to gdoc.../